pkg/server/v2/server.go

```diff
@@ -84,6 +92,22 @@ func (c CallbackFuncs) OnStreamClosed(streamID int64) {
 	}
 }
 
+// OnDeltaStreamOpen invokes StreamOpenFunc.
```

Comment:
These comments aren't updated to say that it invokes the delta variant

In general these comments feel pretty redundant, I wouldn't mind getting rid of them (unless we have some godoc integration that really wants these)

Reply:
Yup updated... I kept these for consistencies sake. I'm open to heaving them but I wanted the code to look familiar to what as already in SOTW